kpatch-build/kpatch-build

```diff
@@ -41,7 +41,7 @@ SCRIPTDIR="$(readlink -f "$(dirname "$(type -p "$0")")")"
 ARCH="$(uname -m)"
 CPUS="$(getconf _NPROCESSORS_ONLN)"
 CACHEDIR="${CACHEDIR:-$HOME/.kpatch}"
-SRCDIR="$CACHEDIR/src"
+KERNEL_SRCDIR="$CACHEDIR/src"
```

Comment:
This SRCDIR/KERNEL_SRCDIR split is unfortunate, and is most likely going to be a future source of bugs (but I don't have any better ideas).

At the very least, a comment would help to describe the difference between the two for future code readers trying to make sense of it.

Reply:
How about we rename the leftover `SRCDIR` to `BUILDDIR`? I think it will better convey it's purpose and then `KERNEL_SRCDIR`/`OOT_MODULE_SRCDIR` are easier to understand as well.

Reply:
Yes, that's much better.  Then there would be no comment needed since the variables are more self-descriptive.